Fails to build without Xwayland since 0.3

Regressed by #132. From [error log](https://github.com/WillPower3309/swayfx/files/12206162/swayfx-0.3.2.log):
```c
sway/tree/root.c:63:7: error: implicit declaration of function 'wlr_surface_is_xwayland_surface' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
                if (wlr_surface_is_xwayland_surface(con->view->surface)) {
                    ^
sway/tree/root.c:63:7: note: did you mean 'wlr_surface_is_layer_surface'?
/usr/include/wlr/types/wlr_layer_shell_v1.h:144:6: note: 'wlr_surface_is_layer_surface' declared here
bool wlr_surface_is_layer_surface(struct wlr_surface *surface);
     ^
sway/tree/root.c:64:44: error: implicit declaration of function 'wlr_xwayland_surface_from_wlr_surface' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
                        struct wlr_xwayland_surface *xsurface = wlr_xwayland_surface_from_wlr_surface(con->view->surface);
                                                                ^
sway/tree/root.c:64:44: note: did you mean 'wlr_xdg_surface_from_wlr_surface'?
/usr/include/wlr/types/wlr_xdg_shell.h:494:25: note: 'wlr_xdg_surface_from_wlr_surface' declared here
struct wlr_xdg_surface *wlr_xdg_surface_from_wlr_surface(
                        ^
sway/tree/root.c:64:33: error: incompatible integer to pointer conversion initializing 'struct wlr_xwayland_surface *' with an expression of type 'int' [-Werror,-Wint-conversion]
                        struct wlr_xwayland_surface *xsurface = wlr_xwayland_surface_from_wlr_surface(con->view->surface);
                                                     ^          ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
sway/tree/root.c:65:4: error: implicit declaration of function 'wlr_xwayland_surface_set_minimized' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
                        wlr_xwayland_surface_set_minimized(xsurface, minimize);
                        ^
```
Steps to reproduce:
```
$ git clone -b 0.16.1 https://gitlab.freedesktop.org/wlroots/wlroots.git subprojects/wlroots
$ meson setup --force-fallback-for=wlroots -Dxwayland=disabled _build
$ meson compile -C _build
````

--- a/sway/tree/root.c
+++ b/sway/tree/root.c
@@ -58,12 +58,16 @@ void root_destroy(struct sway_root *root) {
 /* Set minimized state from scratchpad container `show` state */
 static void root_scratchpad_set_minimize(struct sway_container *con, bool minimize) {
 	if (con->view) {
-		struct wlr_foreign_toplevel_handle_v1 *foreign_toplevel = con->view->foreign_toplevel;
-
+#if HAVE_XWAYLAND
 		if (wlr_surface_is_xwayland_surface(con->view->surface)) {
-			struct wlr_xwayland_surface *xsurface = wlr_xwayland_surface_from_wlr_surface(con->view->surface);
+			struct wlr_xwayland_surface *xsurface
+				= wlr_xwayland_surface_from_wlr_surface(con->view->surface);
 			wlr_xwayland_surface_set_minimized(xsurface, minimize);
-		} else if (foreign_toplevel) {
+			return;
+		}
+#endif
+		struct wlr_foreign_toplevel_handle_v1 *foreign_toplevel = NULL;
+		if ((foreign_toplevel = con->view->foreign_toplevel)) {
 			wlr_foreign_toplevel_handle_v1_set_minimized(foreign_toplevel, minimize);
 		}
 	}

